run_oscar.json

```diff
@@ -0,0 +1,107 @@
+{
+    "id": "run_oscar",
+    "summary": "Run an external OSCAR service.",
+    "description": "Run an external docker-based OSCAR service.",
+    "categories": [
+        "oscar",
+        "service"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "token_env_var",
```

Comment:
Token handling should be performed internally with predefined env variables, no need to have a specific parameter here.